src/pipeline/scraper.py

```diff
@@ -0,0 +1,100 @@
+import logging
+import os
+import time
+
+from dotenv import load_dotenv
+from firecrawl import Firecrawl
+
+load_dotenv()
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+def clean_headnote(text: str) -> str:
+    """
+    Removes headnote from scraped content from website
+
+    :param text: text to clean
+    :type text: str
+    :return: cleaned text
+    :rtype: str
+    """
+
+    marker = "![](https://ww2.mini.pw.edu.pl/wp-content/uploads/WMiNI-01.png)"
+    text = text.split(marker, 1)[1]
+
+    return text
+
+
+def clean_footnote(text: str) -> str:
+    """
+    Removes footnote from scraped content from website
+
+    :param text: text to clean
+    :type text: str
+    :return: cleaned text
+    :rtype: str
+    """
+
+    marker = "#### Zaloguj się"
+    text = text.split(marker, 1)[0]
+
+    return text
+
+
+def scrap_data() -> dict:
+    """
+    Scrap content from mini website.
+    For now only for 15 links.
+
+    :return: Description
+    :rtype: dict
+    """
```

Comment:
Piszmy docstringi w googlowskim stylu + samo dict zdecydowanie za mało tłumaczy.
```suggestion
def scrap_data() -> dict[str, dict[str, str | list]]:
    """
    Scrape content from mini website. 
    For now only for 15 links. 

    Returns:
        dict[str, dict[str, str | list]]:  Dictionary mapping URLs to their scraped content.
            Structure: {
                url (str): {
                    "text" (str): Cleaned markdown content,
                    "links" (list): List of links found on the page
                }
            }
    """
```